src/app/client/src/app/modules/notes/components/note-card/note-card.component.ts

```diff
@@ -206,8 +206,11 @@ export class NoteCardComponent implements OnInit, OnChanges {
   public viewAllNotes() {
     this.activatedRoute.params.subscribe(params => {
       this.batchId = params.batchId;
-      if (this.batchId) {
-        this.route.navigate(['/learn/course', this.ids.courseId, 'batch', this.batchId, 'notes']);
+      if (this.batchId && this.activatedRoute.snapshot.queryParams.contentId) {
```

Comment:
this.activatedRoute.snapshot.queryParams.contentId replace it with _.get(this.activatedRoute, 'snapshot.queryParams.contentId')